docs/feature.avsc

```diff
@@ -71,4 +79,4 @@
     },
 		{"name": "quantmsio_version", "type": "string", "doc": "The version of quantms.io"}
```

Comment:
I think if we want to make consistency, yes, each raw should carry that. We also include it in the project.json, but if someone takes a parquet independently, we will lose track of the schema version.

Reply:
@timosachsenberg you are right, this is not needed, I have been exploring and Parquet allows to storing metadata of the file as key-value pair, this can be done at the level of the field or the file: 

```
my_schema = pa.schema([
    pa.field("movie", "string", False, metadata={"spanish": "pelicula"}),
    pa.field("release_year", "int64", True, metadata={"portuguese": "ano"})],
    metadata={"great_music": "reggaeton"})
```